search-app/src/cmr/search/api/autocomplete.clj

```diff
@@ -0,0 +1,63 @@
+(ns cmr.search.api.autocomplete
+  "Defines the API for autocomplete-suggest in the CMR."
+  (:require
+    [clojure.string :as str]
+    [cmr.common.log :refer (debug info warn error)]
+    [cmr.common.services.errors :as svc-errors]
+    [cmr.search.services.autocomplete-service :as ac]
+    [cmr.search.services.query-service :as qs]
+    [cmr.common-app.api.routes :as common-routes]
+    [cmr.common.mime-types :as mt]
+    [compojure.core :refer :all]))
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+;;; Support Functions
+
+(def limit-default 20)
+(def offset-default 0)
```

Comment:
CMR uses `page_size` and `page_num` instead of `limit` and `offset`. To keep things consistent i think we should use those here. See: `cmr.common-app.services.search.parameter-validation`